docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md

```diff
@@ -0,0 +1,205 @@
+
+This package consists of a series of dbt models that produce the following tables:
+
+- `snowplow_attribution_paths_to_conversion`: Customer id and the the paths the customer has followed that have lead to conversion
+- `snowplow_attribution_campaign_attributions`: By campaign path and conversion level incremental table that attributes the conversion value based on various algorythms
+- `snowplow_attribution_channel_attributions`: By channel path and conversion level incremental table that attributes the conversion value based on various algorythms
+- `snowplow_attribution_overview`: The user defined report view (potenitally showing ROIs)
+- `snowplow_attribution_path_summary`: For each unique path, a summary of associated conversions, optionally non-conversions and revenue
+- `snowplow_attribution_paths_to_non_conversion`: Customer id and the the paths the customer has followed that have not lead to conversion
+
+In the [Quick Start](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/index.md) section you will find a step-by-step guide on how to operate the package as a whole.
+
+## What is Marketing Attribution?
+
+Marketing attribution determines which marketing tactics are contributing to sales or conversions by analysing the marketing touchpoints a consumer encounters on their journey to purchase. The aim is to determine which channels and marketing campaigns had the greatest impact on the decision to convert. There are many popular attribution models used by marketers which give insight into customers' behaviours, more specifically the pathways they took to purchase the product or service. This allows marketing teams to improve ROI by changing marketing strategies and campaigns.
+
+## How to do Marketing Attribution with Snowplow?
+
+In the past have taken Google's attribution model as a base (fractribution) and converted it into a dbt based tool. We considered the dbt-snowplow-fractrubtion package as an exploratory tool for ad-hoc analysis. The data was modelled then the more complex attribution analysis was run on python for efficiency.
+
+Due to customer demand we have also created this package, where we are taking this a step further as the data model can be run on a continuous basis so that the attribution analysis can be done any time in a cost effective and timely manner without the need of python for several marketing attribution algorythms.
+
+In the below guide we will walk you through the data transformation process step-by-step in order for your to see how the source data changes downstream to give you and your team a transparent and easy-to-understand way to understand how this package will get you the valuable insights.
```

Comment:
We can discuss with John, perhaps a blog post will be enough to clear the differences but I feel like we need to explain what is happening in the background, otherwise people might be confused.

Reply:
Yeah blog post makes sense, and the release announcement

Reply:
John said blog post will come together with the data app, till then let's do the release announcement right